Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,237 @@
+
+The guidance is, of course, to always use the beta channel for the script, and this is what the script will have as the default option. 
+
+### Installation script features
+The following arguments are needed for the installation script:
+
+>--channel / -Channel
+>Which channel (i.e. "dev", "beta", "release") to install from; defaults to beta if not present. 
+
+>--version / -Version
+>Which version of CLI to install; you need to specify the version as 3-part version (i.e. 1.0.0-13232323); defaults to latest if not present. 
+
+>--prefix / -InstallDir
+>Path to where to install the CLI bundle; defaults to /.dotnet if not present.
+
+**TODO: how do you call the script**
+
+### Windows command example 
+
+```
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.ps1'))}"
+```
+
+### OSX/Linux shell command
+
+```
+curl -sSL https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/install.sh | bash /dev/stdin [args] 
+```
+
+## Complete manual installation
+Same bits as the previous install way, just that instead of the script, the user downloads the zip/tarball manually and then points the path and whatnot. 
+
+This install covers the following scenarios: 
+
+* Local installation 
+
+The scenarios are completely the same as with the install script.  
+
+### Installation example for manual installation
+
+This example is using OS X. Windows and Linux are similar, the commands may be different but the flow is the same. 
+
+1. Download the tarball from a stable location
+
+```console
+curl -so cli_tarball.tar.gz https://url.org/to/cli/tarball
+```
+
+2. Extract into the needed location
+
+```console
+tar -xf cli_tarball.tar.gz
+```
+
+3. Run the tools
+```console
+./bin/dotnet build
+```
+
+4. At this point, the local installation is complete. The tools can be ran from the directory where they are extracted. 
+
+5. Point the system-wide path to extracted package
+
+```console
+export PATH=$PATH:/path/to/dotnet:/path/to/dotnet/bin
+```
+
+6. At this point, the global install is done. You can access that version of CLI tools from any place in the shell. 
+
+## Docker 
+[Docker](https://docs.docker.com/) has become a pretty good way to use developer tools, from trying them out in an interactive image use to using it for deployment. We have Docker images on DockerHub already. 
+
+Docker images should always be updated as we make new releases. We should have Docker images of stable releases, built from the rel/* branches. 
+
+## NuGet packages
+**TODO: add stuff around how to version properly to not get into wrong channel by accident.**
+
```

Comment:
we shoud have a targetted doc that says "How to use a channel". Different pivot on the same data.